crates/jarscanner/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use clap::Parser;
+use std::collections::HashMap;
+use std::error::Error;
+use std::path::PathBuf;
+
+use bzl_gen_jarscanner as jarscanner;
+
+#[derive(Parser)]
+#[command(author, version, about)]
+#[command(version = "0.1.0")]
+#[command(about = "Jar to containing classes tool", long_about = None)]
+struct Opt {
+    #[arg(long)]
+    out: String,
+
+    #[arg(long)]
+    label: String,
+
+    #[arg(long)]
+    input_jar: String,
+
+    #[arg(long)]
+    relative_path: String,
+}
+
+#[tokio::main]
```

Comment:
a bit weird to use tokio but no other async right? Do we see a path to use async? I'm not sure if you ever would on a single file. If we took a list of files to scan, yes.

Reply:
Yeah a little bit of pattern matching with what our other CLI tools do here. I'll remove